### PR TITLE
Show-MeditationPrompt.Data.psd1: Expand wisdom flavour text collection

### PR DESCRIPTION
**Pull request by Maamue**
# PR Summary

Expand wisdom flavour text collection as asked in #290 

## Changes

Some new flavor texts. We can edit or remove any of them, those are some I thought may fit your request

## Checklist

- [x] Pull Request has a meaningful title.
- [x] Summarised changes.
- [x] Pull Request is ready to merge & is not WIP.
- [ ] Added tests / only testable interactively.
  - Make sure you add a new test if old tests do not effectively test the code changed.
- [ ] Added documentation / opened issue to track adding documentation at a later date.

### CODE REVIEW

File: PSKoans/Data/Show-MeditationPrompt.Data.psd1 [hunk 1]
```diff
@@ -192,6 +192,39 @@ of your own catastrophe.
...
+Do not dwell in the past,
+do not dream of the future,
+concentrate the mind on the present moment.
+'@
+        @'
+Never fear the unknown,
+thus the unknown does not fear you.
+'@
+        @'
+Possibilities are here, there, and wverywhere.
```
Comment (vexx32) [inline PSKoans/Data/Show-MeditationPrompt.Data.psd1:213]:
```suggestion
Possibilities are here, there, and everywhere.
```

File: PSKoans/Data/Show-MeditationPrompt.Data.psd1 [hunk 2]
```diff
@@ -192,6 +192,39 @@ of your own catastrophe.
...
+Keep your mind open to even more.
+'@
+        @'
+The most life gives us,
+often comes from failure at first.
+'@
+        @'
+When it gets tough,
+keep believeing that
```
Comment (vexx32) [inline PSKoans/Data/Show-MeditationPrompt.Data.psd1:222]:
```suggestion
keep believing that
```

File: PSKoans/Data/Show-MeditationPrompt.Data.psd1 [hunk 3]
```diff
@@ -192,6 +192,39 @@ of your own catastrophe.
...
+everyhting will change for the better.
```
Comment (vexx32) [inline PSKoans/Data/Show-MeditationPrompt.Data.psd1:223]:
```suggestion
everything will change for the better.
```

**Review by vexx32 — CHANGES_REQUESTED** [review]
Yeah these fit really well, thanks! 😄 

A couple small spelling errors, but otherwise I like 'em. Thanks! 💖

File: PSKoans/Data/Show-MeditationPrompt.Data.psd1 (hunk 3 above)
Comment (Maamue) [reply PSKoans/Data/Show-MeditationPrompt.Data.psd1:223]: okay, now I start to feel embarassed about the amount of typos...

File: PSKoans/Data/Show-MeditationPrompt.Data.psd1 (hunk 2 above)
Comment (Maamue) [reply PSKoans/Data/Show-MeditationPrompt.Data.psd1:222]: yikes, that went totally past me

File: PSKoans/Data/Show-MeditationPrompt.Data.psd1 (hunk 1 above)
Comment (Maamue) [reply PSKoans/Data/Show-MeditationPrompt.Data.psd1:213]: I thought I fixed that one 🤔

**Comment by Maamue** [comment]
This should address all of the typos you pointed out

**Review by vexx32 — APPROVED** [review]
Perfect, thanks! 💖 😊

**Pull request merged**